Example/StripeConnectExample/StripeConnectExample/MainViewController.swift

```diff
@@ -198,7 +202,7 @@ class MainViewController: UITableViewController {
     func presentAlert(title: String, message: String) {
         let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
         alert.addAction(UIAlertAction(title: "OK", style: .default))
-        self.present(alert, animated: true)
+        (presentedViewController ?? self).present(alert, animated: true)
```

Comment:
When this VC is already presenting a component VC modally, the system won't let us present another modal. So instead, check if we're already presenting something modally and stack on top of that, otherwise present on self.